doc/devel/documenting_mpl.rst

```diff
@@ -901,7 +901,7 @@ Some helpful functions::
 
 .. _ReST: http://docutils.sourceforge.net/rst.html
 .. _Sphinx: http://www.sphinx-doc.org
-.. _documentation: http://www.sphinx-doc.org/contents.html
+.. _documentation: https://www.sphinx-doc.org/en/master/contents.html
```

Comment:
Maybe
```suggestion
.. _documentation: https://www.sphinx-doc.org/en/stable/contents.html
```

Reply:
I wondered about that, but https://www.sphinx-doc.org redirects to the `master` version